catroid/src/androidTest/java/org/catrobat/catroid/test/content/bricks/BrickContextMenuTest.kt

```diff
@@ -153,4 +153,13 @@ class BrickContextMenuTest(
     fun testExpectedHelp() {
         assertTrue(contextMenuItems.contains(R.string.brick_context_dialog_help))
     }
+
+    @Test
+    fun testExpectedAddToBackpack() {
+        val showsAddToBackpack: Boolean = when (brick) {
+            is UserDefinedReceiverBrick, is WhenStartedBrick -> contextMenuItems.contains(R.string.backpack_add)
```

Comment:
I think we could add the other Event blocks here to increase the scope of the test.

Reply:
added the other when bricks to the test